larndsim/pixels_from_track_ep.py

```diff
@@ -6,6 +6,7 @@
 
 import eagerpy as ep
 import numpy as np
+import torch
```

Comment:
do we need this?

Reply:
Nope, sorry -- switched over the unique call to torch at one point and forgot to take this out